docs/For Users/Advanced/Test with ChromeDriver.md

```diff
@@ -15,8 +15,7 @@ The following workflow uses [selenium-python](http://selenium-python.readthedocs
 
 ### Installing
 
-* Download ChromeDriver from NW.js website. It's in the SDK build.
-* Extract the package and place `chromedriver` under the same dir that contains the NW.js binaries: `nw` for Linux, `nw.exe` for Windows, or `node-webkit.app` for Mac.
```

Comment:
Why is this removed?

Reply:
I tried the above steps(Copying 'chromedriver.exe' from SDK build to nw folder). But that doesn't seem to be working. Instead, when I built my app with the SDK version of 'nw', it worked.